### DISCUSSION TITLE
Guestbook main discussion

### DISCUSSION THREAD

**Discussion by EliasAfara** [EliasAfara/portfolio#1] [Guestbook]
# Welcome to the discussion section of my GitHub portfolio

I appreciate your presence and engagement in this space. To ensure a pleasant and respectful experience for everyone, I kindly request that you approach discussions with politeness, kindness, and respect.

Please remember that this is a personal platform, and behind every contribution is a person who has taken the time to share their thoughts and ideas. Let's embrace diversity and maintain a positive atmosphere where different perspectives can be exchanged constructively.

Thank you for your understanding and cooperation in creating a supportive environment within my GitHub portfolio.

Best regards,
Elias

> **Reference**
>  - [Giscus App](https://giscus.app/)
>  - [Giscus Component](https://github.com/giscus/giscus-component)
> - [How to setup example](https://blog.greenroots.info/comment-system-using-react-and-github-discussions)

**Comment by EliasAfara** [comment]
Awesome!

**Comment by rafara** [comment]
Good luck 7abibi!

**Comment by RoyNassif** [comment]
Impressive portfolio! Your work reflects a high level of skill, creativity, and dedication.

**Reply by EliasAfara** [reply]
Thanks

**Comment by cherrydub** [comment]
Loving the neutral colours Elias! Very neat site

**Comment by M0E313** [comment]
Awesome portfolio Elias! Can't wait to use the template XD
